myapp/db/migrate/20230718054642_add_role_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddRoleToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :role, :integer
```

Comment:
[FYI]

If you want to control the order when adding columns, you can use the `after` option.

```suggestion
    add_column :users, :role, :integer, after: description
```

https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_column

Reply:
OK! I just realized that new column will be added after the created_at & updated_at which is hard to read when the table has many columns, I will add `after` in later migration scripts